src/mca/pcompress/zlib/compress_zlib.c

```diff
@@ -176,7 +178,7 @@ static bool doit(uint8_t **outbytes, size_t len2, const uint8_t *inbytes, size_t
 }
 static bool zlib_decompress(uint8_t **outbytes, size_t *outlen, const uint8_t *inbytes, size_t inlen)
 {
-    int32_t len2;
+    uint32_t len2;
```

Comment:
Probably need to change the format in line 192 to be `%u` as well.

Reply:
Yes, thanks for highlighting!